tests/test_optim.py

```diff
@@ -151,3 +152,20 @@ def test_optim_uncerts(backend, source, spec, mu):
     )
     assert result.shape[1] == 2
     assert pyhf.tensorlib.tolist(result)
+
+
+@pytest.mark.skip_numpy
+@pytest.mark.skip_numpy_minuit
+def test_autodiff_minimize_failure(backend, source, spec):
+    pdf = pyhf.Model(spec)
+    data = source['bindata']['data'] + pdf.config.auxdata
+
+    init_pars = pdf.config.suggested_init()
+    par_bounds = pdf.config.suggested_bounds()
+
+    optim = pyhf.optimizer
+    with mock.patch('pyhf.optimize.autodiff.minimize') as mocked_minimize:
```

Comment:
This is currently failing (https://github.com/scikit-hep/pyhf/runs/408386678?check_suite_focus=true#step:8:70) as

```pytb
        if not self.create and original is DEFAULT:
            raise AttributeError(
>               "{} does not have the attribute {!r}".format(target, name)
            )
E           AttributeError: <module 'pyhf.optimize.autodiff' from '/home/runner/work/pyhf/pyhf/src/pyhf/optimize/autodiff.py'> does not have the attribute 'minimize'
```